[DOCS] Update feature toggle examples (#1595)
TYPO3-Documentation/TYPO3CMS-Reference-CoreApi: Documentation/ApiOverview/FeatureToggles/Index.rst

Naming of feature toggles
=========================

> Feature names should NEVER be named "enable" or have a negation, or contain versions or years and should be :php:`loweCamelCase`d.

Reviewer: ```suggestion
Feature names should NEVER be named "enable" or have a negation, or contain versions or years.
It is recommended to use `lowerCamelCase` notation for the feature names.
```

Good examples:

> - `extendedRichtextFormat`

Reviewer: Can you add the `lowerCamelCase` suggestion to the initial sentence in this section?

Author: Done, I hope this is what you expected :-)
